marketplace_standard_app_api/models/object_storage.py

```diff
@@ -36,3 +36,17 @@ class DatasetModel(BaseModel):
 
 
 DatasetListResponse = List[DatasetModel]
+
+
+class MappingName(ConstrainedStr):
+    min_length = 1
+    max_length = 255
```

Comment:
@pablo-de-andres Are these constraints based on anything or basically arbitrary?

Reply:
I just based it off your `CollectionName`. But maybe setting only the `min_length`, like on the `DatasetName` would be more appropriate.